package.json

```diff
@@ -23,7 +24,7 @@
     "storybook": "start-storybook -p 6006 -c stories",
     "storybook:build": "cd stories && npm install && cd .. && build-storybook -c stories",
     "setup": "npm install && npm run storybook:build -s && npm run validate",
-    "validate": "kcd-scripts validate lint,build-and-test,test:cover,test:ts,test:ssr",
+    "validate": "kcd-scripts validate lint,build-and-test,test:cover,test:ts,test:ssr,test:native",
```

Comment:
We need to make sure that the `test:native` runs after the build finishes, so let's actually make this part of the `npm run test:build` script. Could we do this?

```javascript
"test:build": "jest --projects other/misc-tests other/react-native"
```

I _think_ that'll work. Might need to make a few changes (like set the `rootDir` in the config (using the `path` module?) rather than via a flag). That way it runs both of the tests in parallel, then we don't need to change the validate script, and we also wouldn't need to add a `test:native` script.

Does that make sense?

Reply:
I think so!

> then we don't need to change the validate script

Reply:
Man, Jest rocks. Using `--projects` works perfectly!